hw/formal/tools/jaspergold/fpv.tcl

```diff
@@ -41,7 +41,7 @@ if {$env(TASK) == "FpvSecCm"} {
     -bbox_m prim_ram_1p \
     -bbox_m prim_ram_1p_scr \
     -f [glob *.scr]
-} elseif {$env(DUT_TOP) == "pinmux_tb"} {
+} elseif {($env(DUT_TOP) == "pinmux_tb") || ($env(DUT_TOP) == "pinmux_chip_tb")} {
```

Comment:
@matutem this change is needed to also black box the usbdev_aon_wake module for the "new" prim_chip_tb target.